src/main/java/core/basesyntax/operations/SupplyOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.FruitTransaction;
+import java.util.Map;
+
+public class SupplyOperation implements OperationHandler {
```

Comment:
The class `SupplyOperation` should be placed in the `strategy` package according to the checklist item: 'Your project structure should consist the following packages: - `strategy` for holding handlers for different operations (you are expected to apply Strategy pattern).' Make sure to move this class to the correct package.